pkg/crcontrollers/reservation/reservationcontroller.go

```diff
@@ -19,30 +23,55 @@ import (
 
 const (
 	contextTimeoutSeconds = 60
+
+	// reservation parameters
+	fastTimeoutEnv = "RESERVATION_FAST_TIMEOUT"
+	slowTimeoutEnv = "RESERVATION_SLOW_TIMEOUT"
+	attemptsNumEnv = "RESERVATION_ATTEMPTS_NUM"
+
+	defaulFastTimeout = 1500 * time.Millisecond
+	defaulSlowTimeout = 12 * time.Second
+	defaulAttemptsNum = 30
 )
 
 // Controller to reconcile aviliablecapacityreservation custom resource
 type Controller struct {
 	client                 *k8s.KubeClient
 	log                    *logrus.Entry
 	capacityManagerBuilder capacityplanner.CapacityManagerBuilder
+	fastTimeout            time.Duration
```

Comment:
The variables should have name like this:
```golang
fastTimeoutMs   = 1500 * time.Millisecond
slowTimeoutSec = 12 * time.Second
```
So the type of duration denoted in the name.

Reply:
Refactored with RateLimiter args names from [go-client](https://github.com/kubernetes/client-go/blob/8f44946f6cbe967fbe2e2548e76987680a89428e/util/workqueue/default_rate_limiters.go#L136)